### ISSUE TITLE
Load Skins/TextureAtlas from file instead of XNB ?

### ISSUE THREAD

**Issue by RUSshy** [nofilenamed/NezCustom#1]
Hello

Im glad i found this fork, i once asked owner of Nez if he could implement loader from file, but he said no he want to stick with the annoying content pipeline

Do you think you can implement that ?

Thanks!

**Comment by nofilenamed** [comment]
Of course. I'll do this in a few days

**Comment by nofilenamed** [comment]
By the way. Which files you want to load from non-xnb file?

**Comment by RUSshy** [comment]
Thanks

I had something for the Skin, but yours is different than original Nez

**Comment by nofilenamed** [comment]
I know, I'll try to clear and optimize this code , to make it faster
For now, there are only a few changes from methods to get; set;
I hope this isn't a problem.
The port of libgdx (java) is cool and a nice idea, but porting means not, to "clone" and re-implement it in the same way.

**Comment by RUSshy** [comment]
Yes you are right, but libGDX is a so nice library :D

**Comment by nofilenamed** [comment]
Better organization and better code is more better, or not? :D
I'll try to keep all similar to libGDX

**Comment by nofilenamed** [comment]
BTW, I need someone who could create more Widgets, like an inventory with slots, more bars and so on,
could you help?

**Comment by RUSshy** [comment]
I made one for a libGDX game that i'm currently porting to nez, i can share it once done yes :)

**Comment by nofilenamed** [comment]
Really? That sounds nice. If I finished the new TiledMapCollisions and the work on the UI, I'll create the load from file immediately.

**Comment by RUSshy** [comment]
Yes but it gonna be a while because i need first to learn correctly ECS, wich is giving me headache :(

**Comment by nofilenamed** [comment]
What is your problem with the ECS?

**Comment by RUSshy** [comment]
Mostly how to get out of OOP and structure my data ^^
